packages/devtools-launchpad/assets/Svg.js

```diff
@@ -1,5 +1,5 @@
 const React = require("react");
-const InlineSVG = require("svg-inline-react");
+import InlineSVG from "svg-inline-react";
```

Comment:
why should we do this ?
In my experience, `import` and `require` don't mix well.

Reply:
To quote the author:

> I removed dist and es and made lib (which is ES2015 source) default, and there are several reason for it. For now use of ES2015 (and modules) is widespread, there are many tools supporting it (i.e. Webpack 2, Rollup), and you will use them anyway – **if you are using React/etc. If you want to stay in CommonJS land, please specify deps as 1.x.x. i.e) svg-inline-react: 1.x.x**

So require no longer works with this package.

The problem is that the 1.x.x versions trigger warnings so we have to use ES6 import.